integration-test/extensionconfig/public-azurechina.json

```diff
@@ -0,0 +1,26 @@
+{
+	"docker" : {
+		"port" : "2376",
+		"options" : [
+			"--label",
+			"foo=bar"
+			]
+	},
+	"compose" : {
+		"envdump" : {
+			"image" : "mirror.azure.cn:5000/library/busybox",
```

Comment:
@ahmetalpbalkan "mirror.azure.cn:5000" is a usual port number for docker private registry, port 443 and 80 are for china local repo, different port num for different usage. And "mirror.azure.cn:5000" is actually https, otherwise must set  --insecure-registry  in options.